docs/maintenance/update-the-devilbox.rst

```diff
@@ -56,6 +56,8 @@ If you want to checkout a specific release tag (such as ``0.12.1``), do a ``git
 
    # Checkout release
    host> cd path/to/devilbox
+   # Ensure you have latest from remote
+   host> git pull
```

Comment:
As mentioned here: https://github.com/cytopia/devilbox/pull/552/files#r268430962

I would change this to `git fetch`